src/applications/vaos/tests/services/healthcare-service/index.unit.spec.js

```diff
@@ -0,0 +1,67 @@
+import { expect } from 'chai';
+import {
+  mockFetch,
+  setFetchJSONResponse,
+  setFetchJSONFailure,
+  mockMultipleApiRequests,
+} from 'platform/testing/unit/helpers';
+
+import { getLocation } from '../../../services/location';
+import { getAvailableHealthcareServices } from '../../../services/healthcare-service';
+import facilities983 from '../../../api/facilities_983.json';
+import clinicList983 from '../../../api/clinicList983.json';
+
+describe('VAOS Healthcare service', () => {
+  before(() => {
+    mockFetch();
+    setFetchJSONResponse(global.fetch, clinicList983);
+  });
+
+  describe('getAvailableHealthcareServices', () => {
+    it('should make successful request', async () => {
```

Comment:
We normally have an error case test, too

Reply:
Done.